SOS/SERV/sos/sos_interface.h

```diff
@@ -0,0 +1,127 @@
+/**
+ * @fileName	:	sos_interface.h
+ * @Created		: 	2:08 AM Tuesday, May 30, 2023
+ * @Author		: 	Hamsters
+ * @brief		: 	Interface file for Simple Operating System 
+ */ 
+
+
+#ifndef SOS_INTERFACE_H_
+#define SOS_INTERFACE_H_
+
+#include "std.h"
+#include "sos_preconfig.h"
+#include "timer_interface.h"

```

Comment:
No need to include this file here. Move this to the sos_program.c file

Reply:
Thanks for noticing, moved the following includes to `sos_private.h` which is only included in the `sos_program.c` ✅ 
```
#include "sos_preconfig.h"
#include "timer_interface.h"
```